src/azure-cli/azure/cli/command_modules/eventhubs/commands.py

```diff
@@ -72,15 +76,14 @@ def load_command_table(self, _):
 
 
 # Namespace Region
+    with self.command_group('eventhubs namespace', custom_command_type=eh_namespace_custom,
+                            is_preview=True) as g:
+        g.custom_command('create', 'create_eventhub_namespace')
```

Comment:
Is it ok to revert `az eventhubs namespace create` from GA back to preview?

Reply:
yes